lib/onetime.rb

```diff
@@ -58,6 +58,7 @@ def load!(mode = nil, _base = Onetime::HOME)
       @locales = OT.load_locales
       @sysinfo ||= SysInfo.new.freeze
       @instance ||= [OT.sysinfo.hostname, OT.sysinfo.user, $$, OT::VERSION.to_s, OT.now.to_i].gibbler.freeze
+      @emailer = OT::SMTPEmailer
```

Comment:
There is a config option actually. We're just lazily not using it here since only the SMTP emailer has setup to do. I've made a note to get back to this after releasing Ruby 3 support. Thanks for highlighting it.